### DISCUSSION TITLE
Context menu not fully visible when creating an annotation at the top of the page

### DISCUSSION THREAD

**Discussion by rhoadsstevens** [ManifoldScholar/manifold#3476] [Accessibility]
Priority level 2 (out of 3) from UW Accessible Technologies

WCAG mapping: 3.2.7 Hidden Controls, 2.4.7 Focus Visible

STEPS TO REPRODUCE:
1. Select a book (i.e. "I My Recall")
2. Turn on caret browsing (insert + F7)
3. Tab into the text
4. Arrow through the text, hold shift as using arrow keys to highlight something on the first line in the text.
5. Observe the context menu when it appears

OBSERVED BEHAVIOR
When highlighting near the top of the page, the upper half of the context menu gets cut off by the top of the screen, so the top few options of the menu are no longer visible.

EXPECTED BEHAVIOR:
The context menu options should always be visible and within the viewport.

**Comment by dananjohnson** [comment]
I'm not able to reproduce this issue. In my testing in Chrome, Firefox, and Safari, the annotation menu consistently displays within the bounds of the viewport.
